lib/tictactoe.rb

```diff
@@ -2,15 +2,17 @@
 require_relative "./presenter"
 require_relative "./board"
 require_relative "./player"
+require_relative "./validator"
 
 class TicTacToe
-  attr_reader :presenter, :display, :board, :player
+  attr_reader :presenter, :display, :board, :players, :validator
 
-  def initialize(presenter, display, board, player)
+  def initialize(presenter, display, board, players, validator)
```

Comment:
This class has quite a few collaborators now. Something to keep an eye on. An ever-growing list of collaborators is often an indication that there may be a simpler design

Reply:
I took out validator as a collaborator